ml/serving/model_io.py

```diff
@@ -0,0 +1,18 @@
+"""
+Module with functions to write/read models to/from different
+locations & data stores.
+"""
+
+import dill as pickle
+from typing import Dict, NoReturn
+
+
+def write_predictor_to_disk(predictor, path) -> NoReturn:
+    with open(path, "wb") as f:
+        pickle.dump(predictor, f)
+
+
+def read_predictor_from_disk(path) -> Dict:
```

Comment:
i wonder if we can use some lib to cache the result of this to disk like this:

https://pypi.org/project/anycache/